### DISCUSSION TITLE
CS%20General%40NEU/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#100] [Announcements]
# CS%20General%40NEU/

https://csmsapp.github.io/CS%20General%40NEU/

**Comment by andyxxchen** [comment]
想问下23fall的DDL是什么时候

**Reply by PinkGlove** [reply]
4月15
https://www.khoury.northeastern.edu/apply/masters-apply/

**Reply by andyxxchen** [reply]
据说越早申请越好？

**Reply by RuijinShen** [reply]
据说coop被取消了。

**Reply by PinkGlove** [reply]
请问coop被取消的说法是如何而来？

**Reply by yyong120** [reply]
这里好像说了“疑似”。
https://www.1point3acres.com/bbs/thread-865235-1-1.html

**Reply by Water3000** [reply]
25fall的中国区info session明确提到coop是NEU的一大优势, 不认为有取消的可能.

**Comment by ZzzeLoong** [comment]
语言分降低成托福92，雅思7了

**Comment by pzy2003** [comment]
25fall gpa2.78录了
